### ISSUE TITLE
Cannot import collections/environments in 7.9.0 without being a member of a workspace

### ISSUE THREAD

**Issue by BrambleRamble** [a85/POSTMan-Chrome-Extension#7463]
My team's standard workflow in Postman is to import/export collections and environment files, which we manage separately according to our more general development process.

As of Postman 7.9.0, using the Import button or importing through the Manage Environments dialog opens a 'Only members of a workspace can perform this action' dialog:
![image](https://user-images.githubusercontent.com/40172442/66648120-ad2d9680-ec22-11e9-9c9e-43f6bd30513f.png)
We don't have a workspace, and, for our workflow, we don't want one. Therefore we cannot view or edit our collections and environments. I have not found a workaround for this, but if anyone can share one it would be greatly appreciated. I will try downgrading for the moment.

I am on Windows 10.

**Comment by preethammavin** [comment]
@BrambleRamble for you to view the environment you need to be in a workspace, saying that there is a default personal workspace every user is a part of if you are trying to import into that it will import the collection/environment. This is usually shown when you are working in a team workspace, for which you need to be a part of it import/create/edit data in that workspace.

I would want to understand the workflow a bit more to understand the issue.

**Comment by sankalp0o** [comment]
@BrambleRamble 

*A bit about Workspaces*
Every user has a default workspace when they start using Postman called "My Workspace". They can additionally create more free personal as well as team workspaces. Workspaces help organise environments and collections by different contexts.

*Possible solution to your problem*
Now from what I understand, you or someone in your team created a workspace that you tried switching to. You can again switch to your older workspace by using the workspace dropdown on the header in the centre. Look for a "My Workspace" in the dropdown, that might be the workspace you have been using all this while.

Please let me know if this doesn't solve your problem, happy to help further! I'm closing this issue for the moment since this looks like an expected behaiour, but I'd still like to hear more details. Specially if you are not able to continue doing your work.

**Comment by BrambleRamble** [comment]
Sorry for the delay in replying, I was offline over the weekend.

@preethammavin Our workflow uses Postman as a viewer, editor and ad-hoc runner, but we version the collections and environments in the same source control as our other code, which is why the importing/exporting is necessary. I don't think I've ever touched the workspace settings before today, since I don't need a team one and generally import/export/delete as needed (hence my obliviousness to the "My Workspace" default!). I don't sign in either.

@sankalp0o I switched back to 7.8.0 on Friday, and have installed 7.9.0 again just now. "My Workspace" is selected at the top, and the import dialogs are working as usual. I suspect that, on my initial upgrade to 7.9.0, my existing/default configuration got messed up so that "My Workspace" got forgotten somehow, causing the import issues that I originally reported. Like I say, I usually leave workspaces well alone, so I don't think it's something I actively changed. The only possible factor I can think of is that I had no collections or environments open at the time of the upgrade, which is probably a bit unusual.

If anyone else runs into this problem then switching to 7.8.0 then 7.9.0 seems to have sorted it out for me; I've hopped back and forth a few times this morning and haven't reproduced the original issue. It doesn't solve the mystery but it does resolve the problem :-)
